poseidon/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I think we should discard on child's license files, there is no reason for them to survive